### PR TITLE
Add information regarding GDPR

### PR DESCRIPTION
**Pull request by julienmathis**

### CODE REVIEW

File: doc/en/faq/gdpr_compliance.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+---------------------------------------
+
+For each MSP's customer, the Centreon Central Server stores in its SQL database the identification information of the users that can access the monitoring service:
+
+* name
+* alias (login), password
+* email address
+* phone number (optional, for notification purpose)
+
+The Central Server also stores the service parameters each users:
```
Comment (sc979) [inline doc/en/faq/gdpr_compliance.rst]: parameters each users -> parameters of each user

File: doc/fr/faq/gdpr_compliance.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+---------------------------------------
+
+For each MSP's customer, the Centreon Central Server stores in its SQL database the identification information of the users that can access the monitoring service:
+
+* name
+* alias (login), password
+* email address
+* phone number (optional, for notification purpose)
+
+The Central Server also stores the service parameters each users:
```
Comment (sc979) [inline doc/fr/faq/gdpr_compliance.rst]: parameters of each user

**Comment by lpinsivy** [comment]
I will translate FR part

File: doc/en/faq/gdpr_compliance.rst [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+---------------------------------------
+
+For each MSP's customer, the Centreon Central Server stores in its SQL database the identification information of the users that can access the monitoring service:
+
+* name
+* alias (login), password
+* email address
+* phone number (optional, for notification purpose)
+
+The Central Server also stores the service parameters parameters of each user:
```
Comment (sc979) [inline doc/en/faq/gdpr_compliance.rst]: parameters x2

**Comment by lpinsivy** [comment]
commit squashed and translate GDPR in French

**Pull request merged**